### PR TITLE
Add troubleshooting for vm start button is not visible

### PR DESCRIPTION
**Pull request by w13915984028**
Issue: https://github.com/harvester/harvester/issues/4659

Add analysis and trouble-shooting

### CODE REVIEW

**Review by jillian-maroket — COMMENTED** [review]
Let me know if the changes work.

**Review by jillian-maroket — APPROVED** [review]
LGTM. Thanks!

File: docs/troubleshooting/vm.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+sidebar_position: 5
+sidebar_label: VM
+title: "VM"
+---
+
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/v1.1/troubleshooting/vm"/>
+</head>
+
+The following sections contain information useful in troubleshooting issues related to troubleshoot Harvester VM management.
```
Comment (markhillgit) [inline docs/troubleshooting/vm.md:11]:
```suggestion
The following sections contain information useful in troubleshooting issues related to Harvester VM management.
```

**Review by markhillgit — APPROVED** [review]
looks good, thanks

File: docs/troubleshooting/vm.md [hunk 2]
```diff
@@ -0,0 +1,245 @@
+---
+sidebar_position: 5
+sidebar_label: VM
+title: "VM"
+---
+
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/v1.1/troubleshooting/vm"/>
```
Comment (bk201) [inline docs/troubleshooting/vm.md:8]: do we need to change the URL here? (and version v1.2's URL)

Comment (w13915984028) [reply docs/troubleshooting/vm.md:8]: From the `Deploy Preview`, it seems the version value  `v1.1` here does not affect. But please @LucasSaintarbor @jillian-maroket  double check, thanks.

Comment (LucasSaintarbor) [inline docs/troubleshooting/vm.md:8]:
```suggestion
  <link rel="canonical" href="https://docs.harvesterhci.io/v1.2/troubleshooting/vm"/>
```

File: versioned_docs/version-v1.1/troubleshooting/vm.md [hunk 3]
```diff
@@ -0,0 +1,245 @@
+---
+sidebar_position: 5
+sidebar_label: VM
+title: "VM"
+---
+
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/v1.1/troubleshooting/vm"/>
```
Comment (LucasSaintarbor) [inline versioned_docs/version-v1.1/troubleshooting/vm.md:8]:
```suggestion
  <link rel="canonical" href="https://docs.harvesterhci.io/v1.2/troubleshooting/vm"/>
```

File: versioned_docs/version-v1.2/troubleshooting/vm.md [hunk 4]
```diff
@@ -0,0 +1,245 @@
+---
+sidebar_position: 5
+sidebar_label: VM
+title: "VM"
+---
+
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/v1.1/troubleshooting/vm"/>
```
Comment (LucasSaintarbor) [inline versioned_docs/version-v1.2/troubleshooting/vm.md:8]:
```suggestion
  <link rel="canonical" href="https://docs.harvesterhci.io/v1.2/troubleshooting/vm"/>
```

**Review by LucasSaintarbor — COMMENTED** [review]
@w13915984028 Updated the canonical links for this new page (https://github.com/harvester/docs/pull/471/files#r1382713672) to v1.2. This tells search engines to point to the v1.2 version of the docs. LGTM otherwise, thanks!

**Review by bk201 — APPROVED** [review]

**Pull request merged**